fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/steps/FintechStagesUtils.java

```diff
@@ -0,0 +1,31 @@
+package de.adorsys.fintech.tests.e2e.steps;
+
+import io.restassured.RestAssured;
+import io.restassured.specification.RequestSpecification;
+
+import java.util.UUID;
+
+public class FintechStagesUtils {
+
+    public static final String X_REQUEST_ID = "X_REQUEST_ID";
+    public static final String PIN = "1234";
+    public static  String USERNAME = "bob";
+    public static final String X_XSRF_TOKEN = "xsrfToken";
+    public static final String X_XSRF_TOKEN_VALUE = "441c1f6f-1bda-43c0-91d2-4b41a5d53a9c";
+    public static final String SESSION_COOKIE = "sessionCookie";
+    public static final String SESSION_COOKIE_VALUE = "%7B%22fintechUserId%22%3A%226164%22%2C%22hashedXsrfToken%22%3A1713814359%7D";
+    public static final String BANK_ID = "BANK_ID";
+    public static final String BANK_ID_VALUE = "53c47f54-b9a4-465a-8f77-bc6cd5f0cf46";
+    public static final String FINTECH_LOGIN_ENDPOINT = "login";
+    public static final String ACCOUNT = "account/";
+    public static final String ANTON_BRUECKNER_ID = "cmD4EYZeTkkhxRuIV1diKA";
+    public static final String MAX_MUSTERMAN_ID = "oN7KTVuJSVotMvPPPavhVo";
+    public static final String BANKSEARCH = "adorsys xs2a";
+    public static final String BANKPROFILE_ENDPOINT = "https://obg-dev-fintechui.cloud.adorsys.de/bank/";
```

Comment:
Instead of hardcoded value use Autowired value from config like this:
```
@Autowired
private SmokeConfig config;

... somehere in code...
config.getAspspProfileServerUri(); // this is ASPSP_PROFILE_BASE_URI
```

(the value will be read from application.yml by Spring)